cmd/fyne_demo/tutorials/data.go

```diff
@@ -140,7 +148,8 @@ var (
 	TutorialIndex = map[string][]string{
 		"":            {"welcome", "canvas", "animations", "icons", "widgets", "collections", "containers", "dialogs", "windows", "binding", "advanced"},
 		"collections": {"list", "table", "tree"},
-		"containers":  {"apptabs", "border", "box", "center", "grid", "split", "scroll"},
+		"containers":  {"border", "box", "center", "grid", "split", "scroll", "tabs"},
+		"tabs":        {"apptabs", "doctabs"},
```

Comment:
See comment in the PR conversation about whether this extra hierarchy is needed

Reply:
Done